### ISSUE TITLE
requirements.txt in docs/ lists AeoLiS v2.0.0.dev2 as a dependency while the latest stable is v2.1.1

### ISSUE THREAD

**Issue by niketagrawal** [openearth/aeolis-python#121]
The requirements.txt file in `docs/` contains the dependencies required to build the documentation and generate rendered html pages locally, however, installing the dependencies using this file would overwrite the existing installation of AeoLiS with the older version.

To check: Is Read the docs using `docs/requirements.txt` for installing dependencies to build the documentation? If yes, `AeoLiS v2.0.0.dev2` in `docs/requirements.txt` should be replaced with `AeoLiS`

**Comment by manuGil** [comment]
According to RTD, the `requirements.txt` is used to pass dependencies that are used by their platform to build the documentation. It is not expected that we will be using that file in any other case. However, we probably want to point out the latest research always.  

Heres's an explanation of the purpose of that file: https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html

**Comment by manuGil** [comment]
Read The Docs will make the use of a configuration file compulsory in September. I added it to the the **docs/build** branch already. I will be merge along with other fixes.

> The Read the Docs build system will start requiring a configuration file v2 (.readthedocs.yaml) starting on September 25, 2023. We are sending weekly notifications about this issue to all impacted users, as well as temporary build failures (brownouts) as the date approaches for those who haven't migrated their projects. The timeline for this deprecation is as follows:
Monday, July 24, 2023: Do the first brownout (temporarily enforce this deprecation) for 12 hours: 00:01 PST to 11:59 PST (noon)
Monday, August 14, 2023: Do a second brownout (temporarily enforce this deprecation) for 24 hours: 00:01 PST to 23:59 PST (midnight)
Monday, September 4, 2023: Do a third and final brownout (temporarily enforce this deprecation) for 48 hours: 00:01 PST to September 5, 2023 23:59 PST (midnight)
Monday, September 25, 2023: Fully remove support for building documentation without configuration file v2.

**Comment by manuGil** [comment]
The reference to equation `equilibrium-transport` in the model description is missing, from the [model.rst](https://github.com/openearth/aeolis-python/edit/main/docs/model.rst)

@Sierd  Could you please check if the equation appears in the file? Maybe the equation is there but it has a differnt name. Or it was moved to another file.
 
```rst
Multi-fraction Erosion and Deposition
-------------------------------------

The formulation for the equilibrium or saturated sediment
concentration :math:`c_{\mathrm{sat}}` (Equation
:eq:`equilibrium-transport`) is capable of dealing with variations in
grain size through the variables :math:`u_{\mathrm{th}}`,
```

**Comment by niketagrawal** [comment]
> @niketagrawal I test compiling the documentation using your branch on decomposition. I still see several issues regarding missing references and loading extensions. Is that the same for you? Shall I fix them?

@manuGil , thanks for checking this. I do see several warnings about incorrect formatting and missing references. Sure, please go ahead with the fix. Thanks again.

**Comment by manuGil** [comment]
> > @niketagrawal I test compiling the documentation using your branch on decomposition. I still see several issues regarding missing references and loading extensions. Is that the same for you? Shall I fix them?
> 
> @manuGil , thanks for checking this. I do see several warnings about incorrect formatting and missing references. Sure, please go ahead with the fix. Thanks again.

@niketagrawal I fixed all of them, except the one about the missing reference to the **equilibrium-transport**.

**Comment by manuGil** [comment]
Equilibrium transport is the Csat equation

**Comment by niketagrawal** [comment]
Closed with #134
